### PR TITLE
Unify bootstrap logic and introduce Runner

### PR DESCRIPTION
**Pull request by stephencelis**
Just a stub executable for now. But wanted to push this plumbing first.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "Sources/Server/Transcripts"]
-	path = Sources/Server/Transcripts
+	path = Sources/PointFree/Transcripts
```
Comment (stephencelis) [inline .gitmodules]: This stuff can live in the `PointFree` module now, which means our bootstrap script can in full, as well.

File: Package.swift [hunk 2]
```diff
@@ -1,15 +1,15 @@
 // swift-tools-version:4.0
 
-import Foundation
```
Comment (stephencelis) [inline Package.swift]: Most of the noise in this file is cleanup.

File: Package.swift [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 import PackageDescription
 
 let package = Package(
   name: "PointFree",
   products: [
-    .executable(name: "Server", targets: ["Server"]),
-    .library(name: "Styleguide", targets: ["Styleguide"]),
     .library(name: "PointFree", targets: ["PointFree"]),
     .library(name: "PointFreeTestSupport", targets: ["PointFreeTestSupport"]),
+    .executable(name: "Runner", targets: ["Runner"]),
```
Comment (stephencelis) [inline Package.swift]: Except for this.

``` sh
$ swift run Runner
```

File: Sources/PointFree/Bootstrap.swift [hunk 4]
```diff
@@ -1,9 +1,11 @@
 import Either
+import Foundation
+import Optics
 import Prelude
 
 public func bootstrap() -> EitherIO<Error, Prelude.Unit> {
-
   return print(message: "⚠️ Bootstrapping PointFree...")
+    .flatMap(const(loadEnvVars))
```
Comment (stephencelis) [inline Sources/PointFree/Bootstrap.swift]: We now load the environment in here!

File: Sources/Runner/main.swift [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import PointFree
```
Comment (stephencelis) [inline Sources/Runner/main.swift]: Runner is so far just a "test" for if the app can boot.

File: Sources/Server/main.swift [hunk 6]
```diff
@@ -1,57 +1,7 @@
-import Foundation
```
Comment (stephencelis) [inline Sources/Server/main.swift]: `Server`'s main file got reeeal short.

File: Sources/PointFree/Bootstrap.swift (hunk 4 above)
Comment (mbrandonw) [reply Sources/PointFree/Bootstrap.swift]: sooooooo much better

File: Sources/Runner/main.swift (hunk 5 above)
Comment (mbrandonw) [reply Sources/Runner/main.swift]: just so I understand, in the future we'd put our "cron" logic in here, like parse arguments passed in from the scheduler to figure out what logic to execute, etc...?

Comment (stephencelis) [reply Sources/Runner/main.swift]: Yeah, we can start with a single task that executes, but we'll eventually wanna use a CLI argument parser. Will be fun to play around with an applicative argument printer/parser.

**Pull request merged**